massa-node/src/main.rs

```diff
@@ -142,9 +142,12 @@ async fn launch(
 
     // Remove current disk ledger if there is one
     // NOTE: this is temporary, since we cannot currently handle bootstrap from remaining ledger
-    if SETTINGS.ledger.disk_ledger_path.exists() {
+    let now = MassaTime::now(0).expect("could not get now time");
+    if *GENESIS_TIMESTAMP > now && SETTINGS.ledger.disk_ledger_path.exists() {
```

Comment:
Why do you not use the keep_ledger flag here? If we are in sandbox, this will always remove the ledger, which we might not want.